### PR TITLE
Add Arch Linux package install instructions

### LINKED ISSUE #294: Missing packages on Arch Linux
On a pretty fresh arch installation, I found the following packages to be missing:

  * `make`
  * `pkg-config`

### PR DESCRIPTION
**Pull request by polyzen**
Includes (relatively) minor touch ups.

Fixes #294.

### CODE REVIEW

**Comment by SolarAquarion** [comment]
alternatively install pacaur and install rust nightly/git

**Comment by colemickens** [comment]
There are AUR packages for this as well.

**Comment by jwilm** [comment]
Does it make sense to just tell arch users to use the AUR package?

**Comment by colemickens** [comment]
I didn't realize, the code change IS using the AUR, just manually building from AUR rather than using a helper. I had seen `makepkg` and assumed it was a PKGBUILD in the `alacritty` tree. Probably fine as is; maybe a link and sentence about it being on the AUR would be helpful.

**Comment by jwilm** [comment]
Does the AUR package have to be updated regularly or does it track this git project automatically?

I'm obviously not familiar with Arch. Thanks for entertaining my questions :)

**Comment by colemickens** [comment]
It's pretty straight forward, here's a quick run down. This is latest PKGBUILD. It's the only thing needed to build the Arch package: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=alacritty-git

There's no direct reference to a specific git version (other than the `pkgver` being `361` which is a nod to the number of commits on master). But the way the `pkgver` function works there... it will actually build from `master` and set the package version to the Cargo.toml verion with the commit count appended.

So, if you were to install it right now, you'd get the current latest commit. It wouldn't be triggered for an auto-update until the owner manually bumps the pkgver or you decide to force re-install.

**Comment by jwilm** [comment]
@colemickens thanks for the explanation--that's actually pretty straight forward. It seems that the changes suggested here are probably good to go.

**Comment by SolarAquarion** [comment]
It tracks it automatically

File: README.md [hunk 1]
```diff
@@ -28,11 +28,20 @@ binaries will be provided for supported operating systems.
...
 
-The only supported installation method at this time is from source. Proper
-installers will be added prior to the 1.0 release of Alacritty. This section
-will walk you through how to build from source on both macOS and Ubuntu.
+The only supported installation method at this time is from source. Proper installers will be added prior to the 1.0 release of Alacritty.
+
+### Arch Linux
+
+```sh
+git clone https://aur.archlinux.org/allacrity-git.git
```
Comment (karlek) [inline README.md]: Misspelled `alacritty`.

File: README.md [hunk 2]
```diff
@@ -28,11 +28,20 @@ binaries will be provided for supported operating systems.
...
+cd allacrity-git
```
Comment (karlek) [inline README.md]: Misspelled `alacritty`.

Comment (polyzen) [inline README.md]: Fixed. :)

**Comment by munyari** [comment]
I think it is sufficient to link to the AUR package, rather than stepping the user through the process of downloading and building with `makepkg`. An Arch Linux user will likely know how to install from the AUR (if they want to). If they do not know, they should definitely refer to the [wiki](https://wiki.archlinux.org/index.php/Arch_User_Repository)

**Pull request merged**